asv.conf.json

```diff
@@ -3,18 +3,11 @@
   "project": "astropy",
   "project_url": "http://www.astropy.org/",
   "repo": "https://github.com/astropy/astropy.git",
-  "build_command": [
-    "python -m build --wheel -o {build_cache_dir} {build_dir}"
+  "install_command": [
+    "pip install . matplotlib scipy"
```

Comment:
What if someone wants to run with different versions of these?

Reply:
[<img width="339" alt="image" src="https://github.com/astropy/astropy-benchmarks/assets/16024299/4812d67c-76ef-4e54-a538-9a35494e8560">](https://github.com/spacetelescope/bench/actions/workflows/benchmark_astropy-benchmarks.yaml)

Reply:
oh yes! Good catch, I'll add that